remo/events/templates/view_event.jinja

```diff
@@ -391,25 +397,38 @@ Mozilla Reps - {{ event.name }}
             </span>
           </p>
           <!-- Listing of organizer and attendees -->
+          {% if event|get_total_attendees >= 3 %}
```

Comment:
Could you change that to `>=5`. Let's switch to an avatar grid when we have more that 4 attendees.